### ISSUE TITLE
[Entrega 2] [grupo impar] Atributo de número de cajas no calza con número de cajas en tabla de cajas

### ISSUE THREAD

**Issue by diegorakos** [IIC2413/Syllabus-2023-1#187]
Hola!

Hay productos cuyo atributo de número de cajas no calza con la cantidad de cajas que tienen en la tabla de cajas. En general, la cantidad de cajas en la tabla son menos que el número indicado por el atributo, y me parece que hay un caso en que son más. ¿Cómo hay que abordar esto? En particular, ¿qué hay que hacer en el contexto de la consulta 3?

Quedo atento.

**Comment by Paldana99** [comment]
Hola!

puede dar un ejemplo, para comprender el problema.

**Comment by diegorakos** [comment]
Claro! Por ejemplo, el producto de `id` 3721. Su atributo `número_cajas` es 1, pero en la tabla de cajas hay dos cajas asociadas al `id_producto` 3721, una con `id` 0 y otra con `id` 1. Creo que ese es el único que le sobra una caja.

Por otro lado, hay varios productos que les faltan cajas, si no fallo la mayoría (quizás todos) de tipo iluminacion. Por ejemplo, el producto de `id` 3723. Su atributo `número_cajas` es 8, pero en la tabla de cajas no hay más de 4 cajas por producto, en particular, este tiene 2, una con `id` 0 y otra con `id` 1.

**Comment by Paldana99** [comment]
Entiendo, en ese caso pueden realizar la corrección que les parezca conveniente. Dejando claro en la entrega como trataron los datos. Por ejemplo pueden re calcular el numero_cajas con lo que aparece en la tabla cajas.
